media/css/pebbles/includes/mixins/_typography.scss

```diff
@@ -48,41 +48,73 @@
 //  .foo { @include font-size-level1; }
 
 @mixin font-size-huge { // For especially huge titles
-    @include font-size(50px);
+    @include font-size(48px);
 
     @media #{$mq-tablet} {
-        @include font-size(90px);
+        @include font-size(60px);
+    }
+
+    @media #{$mq-desktop-wide} {
+        @include font-size(72px);
     }
 }
 
 @mixin font-size-level1 {
-    @include font-size(38px);
+    @include font-size(44px);
 
     @media #{$mq-tablet} {
-        @include font-size(50px);
+        @include font-size(52px);
+    }
+
+    @media #{$mq-desktop-wide} {
+        @include font-size(60px);
     }
 }
 
 @mixin font-size-level2 {
-    @include font-size(28px);
+    @include font-size(32px);
```

Comment:
I'm also open to better names for these mixins if we want to start using them more strictly. The "level" names were inspired by heading levels (h1-h6) so maybe that's good enough. Just sometimes a little weird when we have an h2 explicitly set to level4 or an h4 set to level2.